Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -390,6 +390,27 @@ static const char* environmentVariableValue(const char* name, const char* defaul
     return value ? value : defaultValue;
 }
 
+static void bindUsrMerge(Vector<CString>& args)
+{
+    static const std::array<String, 5> usrMergedDirectories = {
+        "/bin"_s,
+        "/lib"_s,
+        "/lib32"_s,
+        "/lib64"_s,
+        "/sbin"_s
+    };
+    for (auto& directory : usrMergedDirectories) {
+        auto fileType = FileSystem::fileType(directory);
+        if (fileType == FileSystem::FileType::SymbolicLink) {
+            auto symLink = makeString("/usr"_s, directory, '/');
+            args.appendVector(Vector<CString> { "--symlink", directory.utf8(), symLink.utf8() });
```

Comment:
I think this is backwards, which would explain why /usr/bin/xdg-dbus-proxy is missing:

```suggestion
            auto linkTarget = makeString("/usr"_s, directory, '/');
            args.appendVector(Vector<CString> { "--symlink", linkTarget.utf8(), directory.utf8() });
```

Reply:
I tried both ways, didn't notice an improvement either way.